Admin/RouteAdmin.php

```diff
@@ -117,11 +109,6 @@ public function setContentRoot($contentRoot)
         $this->contentRoot = $contentRoot;
     }
 
-    public function setControllerResolver($controllerResolver)
```

Comment:
If you can't remove the setter injection (https://github.com/symfony-cmf/SimpleCmsBundle/blob/1.2.0/Resources/config/admin-phpcr.xml#L36) for BC reasons, I think that adding an empty method (maybe with a deprecated annotation) is the solution.